src/libs/actions/IOU.js

```diff
@@ -135,14 +135,15 @@ function rejectTransaction({
             if (response.jsonCode !== 200) {
                 throw new Error(`${response.code} ${response.message}`);
             }
-            fetchChatReportsByIDs([chatReportID]);
 
-            // If an iouReport is open (has an IOU, but is not yet paid) then we sync the chatReport's 'iouReportID'
-            // field in Onyx, simplifying IOU data retrieval and reducing necessary API calls when displaying IOU
-            // components. If we didn't sync the reportIDs, the transaction would still be shown to users as rejectable
-            // The iouReport being fetched here must be open, because only an open iouReoport can be paid. Therefore,
-            // we should also sync the chatReport after fetching the iouReport.
-            fetchIOUReportByIDAndUpdateChatReport(reportID, chatReportID);
+            // Save the updated chat and IOU reports sent back in the response
+            // NOTE: since the API doesn't handle syncing chat reports with IOU reports,
```

Comment:
NAB grammar
```suggestion
            // Save the updated chat and IOU reports sent back in the response.
            // NOTE: Since the API doesn't handle syncing chat reports with IOU reports,
```